handbook/ops/bizops/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# BizOps onboarding
+
+Welcome! We're so excited to have you join the BizOps team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Week 1
+
+### Get set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
+- [Sourcegraph direction (1 year plan)](../../../direction/index.md)
+- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
```

Comment:
```suggestion
- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
- [CustomerOps project tracker](https://docs.google.com/spreadsheets/d/16YKiI5-jn5xH8t1n7z3pC-4AQfJnOsZLlgJfWj2RlY4/edit#gid=0)
```
@shaan-sourcegraph adding the link to the customer ops project tracker here. Can you make it accessible for all users with a @sourcegraph.com email? Can't currently access it.